package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "clean": "rm -rf src/AWS/Services/*.elm",
-    "build": "npm run clean && node scripts/generate-sdk.js && elm package install -y && elm make --docs=docs.json",
+    "webpack": "./node_modules/.bin/webpack && cat src/Native/HMAC.out.js src/Native/HMAC.src.js > src/Native/HMAC.js",
```

Comment:
`./node_modules/.bin/` can be removed, and it will still work even if webpack is not installed globally